### PR TITLE
[feature ILX- 45346] - Added Latest Set of API's along with changes mentioned during review 

### PR DESCRIPTION
**Pull request by AkshayaKumarIntelex**
!!!!! STOP AND READ !!!!!

If the dropdown above says "base fork: lord/master", you are submitting your change to ALL USERS OF SLATE, not just your company. This is probably not what you want. Click "base fork" to change it to the right place.

If you're actually trying to submit a change to upstream Slate, please submit to our dev branch, PRs sent to the master branch are generally rejected.

### CODE REVIEW

File: source/includes/acts/post_api.md [hunk 1]
```diff
@@ -433,6 +434,91 @@ IRestResponse response = client.Execute(request);
 
 ```
 
+### 5. Equipment Attribute Table 
+
+This section outlines the process of adding new entries or modifying existing records within the Equipment Attribute table using the dedicated API endpoint.
+
+**Equipment Attribute POST Endpoint**
+
+`POST` api/development/v1/equipmentattribute
```
Comment (PriyankaGawade21) [inline source/includes/acts/post_api.md:443]: api should be removed?

File: source/includes/acts/delete_api.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+request(options, function (error, response, body) {
+  if (error) throw new Error(error);
+
+  console.log(body);
+});
+```
+
+```csharp
+var client = new RestClient("https://intelex_url/api/v1/operation");
```
Comment (PriyankaGawade21) [inline source/includes/acts/delete_api.md:60]: api to be removed from URL

**Review by PriyankaGawade21 — APPROVED** [review]

File: source/includes/acts/reference.md [hunk 3]
```diff
@@ -24,10 +24,10 @@ To begin using the Intelex API you will need:
 >Example Requests:
 
 ```CSharp 
-//JSON Body for Authentication Endpoint
+Body for Authentication Endpoint
 {"client_id":"Client ID Data",
 "client_secret":"Client Secret Data",
-"audience":"https://intelex_acts_api_url/api",
+"audience":"https://tenant.actsapi.intelex.com/v1/api",
```
Comment (stevelavoie-pm) [inline source/includes/acts/reference.md:30]: [tenant]

Comment (AkshayaKumarIntelex) [reply source/includes/acts/reference.md:30]: @stevelavoie-pm  the above URL has been changed to https://[tenant].actsapi.intelex.com which is present in the latest commit - 3rd , Thanks .

File: source/includes/acts/get_api.md [hunk 4]
```diff
@@ -156,7 +239,7 @@ Attribute | Type | Description
 --------- | ---- | -----------
 PageNumber | int | Page number of the results to fetch.
 PageSize | int | The number of results per page
-emissionCategoryID | int | Emission Category ID is one of the unique identifier for this emission category record and is one of the filter option provided for Emission Category
+emissionCategoryIDs | int | Emission Category ID is one of the unique identifier for this emission category record and is one of the filter option provided for Emission Category
```
Comment (stevelavoie-pm) [inline source/includes/acts/get_api.md:242]: Emission Category ID is the unique identifier for an emission category record and is one of the filter options provided for Emission Category

File: source/includes/acts/get_api.md [hunk 5]
```diff
@@ -207,7 +302,7 @@ Attribute | Type | Description
 --------- | ---- | -----------
 PageNumber | int | Page number of the results to fetch.
 PageSize | int | The number of results per page
-emissionTypeID | int | Emission Type ID is one of the type of emissions and is the one of the filter option provided for Emission Type
+emissionTypeIDs | int | Emission Type ID is one of the type of emissions and is the one of the filter option provided for Emission Type
```
Comment (stevelavoie-pm) [inline source/includes/acts/get_api.md:305]: Emission Type ID is the unique identifier for an emission type record and is one of the filter options provided for Emission Type

File: source/includes/acts/get_api.md [hunk 6]
```diff
@@ -258,7 +380,15 @@ Attribute | Type | Description
 --------- | ---- | -----------
 PageNumber | int | Page number of the results to fetch.
 PageSize | int | The number of results per page
-equipmentID | int | Equipment ID is one of the unique identifier for this equipment record
+equipmentIDs | int | Equipment ID is one of the unique identifier for this equipment record
```
Comment (stevelavoie-pm) [inline source/includes/acts/get_api.md:383]: Equipment ID is the primary key identifier for equipment records

**Review by stevelavoie-pm — APPROVED** [review]
Approved by PM

**Review by stevelavoie-pm — APPROVED** [review]
Approved by PM

**Pull request merged**